backend/src/main/java/com/tourneynizer/tourneynizer/dao/MatchDao.java

```diff
@@ -104,4 +101,54 @@ public Match findById(Long id) {
         String sql = "SELECT * FROM matches WHERE tournament=?;";
         return this.jdbcTemplate.query(sql, new Object[]{tournament.getId()}, new int[]{Types.BIGINT}, rowMapper);
     }
+
+    public void startMatch(Match match) {
+        String sql = "UPDATE matches SET status=? WHERE id=?";
+        int updated = jdbcTemplate.update(sql, new Object[]{MatchStatus.STARTED.ordinal(), match.getId()},
+                new int[] {Types.SMALLINT, Types.BIGINT});
+        if (updated == 1) {
+            match.setMatchStatus(MatchStatus.STARTED);
+        }
+    }
+
+    public void endMatch(Match match, long score1, long score2) {
+        String sql = "UPDATE matches SET status=?, score1=?, score2=? WHERE id=?;";
+        int updated = jdbcTemplate.update(sql, new Object[]{MatchStatus.COMPLETED.ordinal(), score1, score2, match.getId()},
+                new int[] {Types.SMALLINT, Types.BIGINT, Types.BIGINT, Types.BIGINT});
+
+        if (updated == 1) {
+            match.setMatchStatus(MatchStatus.COMPLETED);
+            match.setScore1(score1);
+            match.setScore2(score2);
+        }
+    }
+
+    public void updateScore(Match match, long score1, long score2) {
+        String sql = "UPDATE matches SET score1=?, score2=? WHERE id=?;";
+        int updated = jdbcTemplate.update(sql, new Object[]{score1, score2, match.getId()},
+                new int[] {Types.BIGINT, Types.BIGINT, Types.BIGINT});
+
+        if (updated == 1) {
+            match.setScore1(score1);
+            match.setScore2(score2);
+        }
+    }
+
```

Comment:
We're also gonna need a getScore at some point (not crazy important)

Reply:
Will do later